app/src/main/res/layout/activity_main.xml

```diff
@@ -25,7 +25,7 @@
         android:layout_marginTop="8dp"
         android:layout_marginEnd="16dp"
         android:layout_marginRight="16dp"
-        android:hint="Search by Furniture"
+        android:hint="@string/search_by_furniture"
```

Comment:
I got this from the Android Studio suggestion bar

Reply:
strings.xml is the best place for string literals.  You can have a different strings.xml for each language you support, and the Android operating system will automatically choose the correct file, based on the user's preferred language.